eng/scripts/Language-Settings.ps1

```diff
@@ -162,18 +162,43 @@ function Get-python-GithubIoDocIndex()
 }
 
 function ValidatePackage($packageName, $packageVersion, $workingDirectory) {
-  $packageExpression = "$packageName$packageVersion"
-  Write-Host "Validating $packageExpression"
-
-  $installTargetFolder = Join-Path $workingDirectory $packageName
-  New-Item -ItemType Directory -Force -Path $installTargetFolder | Out-Null
-
   # Add more validation by replicating as much of the docs CI process as
   # possible
   # https://github.com/Azure/azure-sdk-for-python/issues/20109
+  if (!$ImageId) {
+    Write-Host "Validating using pip command directly on $packageName."
+    FallbackValidation -packageName "$packageName" -packageVersion "$packageVersion" -workingDirectory $workingDirectory
+  } 
+  else {
+    Write-Host "Validating using $ImageId on $packageName."
+    DockerValidation -packageName "$packageName" -packageVersion "$packageVersion"
+  }
+}
+function DockerValidation($packageName, $packageVersion) {
```

Comment:
We should consider moving this to a common location if we start duplicating most of the logic but I'm find with it being hear until we get the other languages in place as well.

Reply:
https://github.com/Azure/azure-sdk-tools/issues/2096